harmonia/src/main.rs

```diff
@@ -125,8 +126,35 @@ async fn main() -> std::io::Result<()> {
     // default is 5 seconds, which is too small when doing mass requests on slow machines
     .client_request_timeout(Duration::from_secs(30))
     .workers(c.workers)
-    .max_connection_rate(c.max_connection_rate)
-    .bind(c.bind.clone())?
-    .run()
-    .await
+    .max_connection_rate(c.max_connection_rate);
+
+    let try_url = Url::parse(&c.bind);
+    let (bind, uds) = {
+        if try_url.is_ok() {
+            let url = try_url.as_ref().unwrap();
+            if url.scheme() != "file" {
```

Comment:
Not exactly sure, but for me `unix` scheme sounds as a more common scheme which many software uses. Why use `file`?

Reply:
Changed.